Remarkable-Donkeys/bangazonCLI
# Kn integrating menus

# Steps To Test
1. `git fetch --all`
2. `git checkout kn-integratingMenus`
3. `cd src`
4. `dotnet run`
5. Add customer to your database if you do not already have a customer in there. When a new customer is created, they will automatically be set as the active customer
6. If you already have  a customer in your database, then select an active customer
7. Add a payment type for that customer
8. Then check the functionality of the Sell Products menu (add, update, delete)

### Description of what should happen when steps are followed
User should be able to add a new payment type for the active customer
User should be able to add, update and delete products for the active customer
User should be able to navigate to Purchase Product menu, but no functionality has been give to the `Add Product` and `Complete Order` options yet

**At least one person needs to approve pull request before merged to master**
